benchmark/python/benchmark_hail/__main__.py

```diff
@@ -1,9 +1,11 @@
+import sys
+
 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
 
 from .run import cli as run
 from . import compare, create_resources, combine, summarize, visualize
 
-def main():
+def main(argv=sys.argv[1:]):
```

Comment:
This change makes it easier for me to launch the benchmark suite in a debugger and change which benchmark i want to debug from within the debugger.